airbyte-webapp/src/components/ui/ListBox/ListBox.tsx

```diff
@@ -2,6 +2,7 @@ import { Listbox } from "@headlessui/react";
 import classNames from "classnames";
 import React from "react";
 
+import CaretDownIcon from "./CaretDownIcon";
```

Comment:
This SVG is exported form the dropdown mockup in [Figma](https://www.figma.com/proto/Xa2bg6dg5AoxA1mFJ3HXsG/MULTI-CLOUD?node-id=374%3A9541&scaling=min-zoom&page-id=374%3A9485&starting-point-node-id=374%3A9486&show-proto-sidebar=1).